engine/src/main/java/org/hibernate/search/v6poc/search/dsl/predicate/impl/MatchAllPredicateContextImpl.java

```diff
@@ -12,50 +12,50 @@
 import java.util.function.Supplier;
 
 import org.hibernate.search.v6poc.search.SearchPredicate;
-import org.hibernate.search.v6poc.search.dsl.predicate.AllPredicateContext;
+import org.hibernate.search.v6poc.search.dsl.predicate.MatchAllPredicateContext;
 import org.hibernate.search.v6poc.search.dsl.predicate.SearchPredicateContainerContext;
 import org.hibernate.search.v6poc.search.dsl.predicate.spi.SearchPredicateDslContext;
-import org.hibernate.search.v6poc.search.predicate.spi.AllPredicateBuilder;
+import org.hibernate.search.v6poc.search.predicate.spi.MatchAllPredicateBuilder;
 import org.hibernate.search.v6poc.search.predicate.spi.BooleanJunctionPredicateBuilder;
 import org.hibernate.search.v6poc.search.predicate.spi.SearchPredicateContributor;
 import org.hibernate.search.v6poc.search.predicate.spi.SearchPredicateFactory;
 
 
-class AllPredicateContextImpl<N, C>
-		implements AllPredicateContext<N>, SearchPredicateContributor<C> {
+class MatchAllPredicateContextImpl<N, C>
+		implements MatchAllPredicateContext<N>, SearchPredicateContributor<C> {
 
 	private final SearchPredicateFactory<C> factory;
 	private final Supplier<N> nextContextProvider;
 
-	private final AllPredicateBuilder<C> builder;
-	private AllExceptContext exceptContext;
+	private final MatchAllPredicateBuilder<C> builder;
+	private MatchAllExceptContext exceptContext;
 
-	AllPredicateContextImpl(SearchPredicateFactory<C> factory,
+	MatchAllPredicateContextImpl(SearchPredicateFactory<C> factory,
 			Supplier<N> nextContextProvider) {
 		this.factory = factory;
 		this.nextContextProvider = nextContextProvider;
-		this.builder = factory.all();
+		this.builder = factory.matchAll();
 	}
 
 	@Override
-	public AllPredicateContext<N> boostedTo(float boost) {
+	public MatchAllPredicateContext<N> boostedTo(float boost) {
```

Comment:
@yrodiere btw, does it make sense to have a boostedTo() for matchAll()?

Reply:
In Elasticsearch it defines the score of that query. Not sure if it's useful, but it seems to make sense, yes.
See https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-match-all-query.html